configure

```diff
@@ -78,6 +78,13 @@ function print_help {
   echo
   echo "  -D var=value                      Specify FLAGS definitions to be passed on to cmake."
   echo
+  echo "  --create-xcode-project            Create an xcode project file as xcode/Turi.xcodeproj."
+  echo
+  echo "  --build-dependencies              Build the external dependencies as part of configure and install them into deps/local."
```

Comment:
what's the relationship between xcode-project with the build-deps?

Reply:
You have to build the dependencies (those not in src/external) separately, as the xcode project does not integrate with their build systems.  So building them means the xcode project then depends on them as lib / include dependencies, but doesn't rebuild them.

Reply:
I see. If I understand correctly, xcode proejct under the hood uses cmakes? And those cannot be integrated with xcode project don't use cmake as their build pipeline?